ui/app/controllers/jobs/index.js

```diff
@@ -342,4 +303,254 @@ export default class JobsIndexController extends Controller {
     }
   }
   //#endregion querying
+
+  //#region filtering and searching
+
+  @tracked statusFacet = {
+    label: 'Status',
+    options: [
+      {
+        key: 'pending',
+        string: 'Status == pending',
```

Comment:
Chose to include string equivalents of what you'd select in your dropdown directly here, rather than have some kind of mapping logic elsewhere